airflow/jobs/triggerer_job.py

```diff
@@ -148,6 +149,25 @@ def load_triggers(self):
         ids = Trigger.ids_for_triggerer(self.id)
         self.runner.update_triggers(set(ids))
 
+    @provide_session
+    def purge_completed_triggers_list(self, session):
```

Comment:
it would be nice if we didn't have to query tthe database here but i don't think it can be avoided.

i explored updating `clean_unused` to return the `ids` deleted.  then we could pass those to `purge_completed` and purge those rows.  but this won't work because `clean_unused` does not filter w.r.t. triggerer_id, so other triggerer instances could delete the rows belonging to _this_ triggerer, so `completed_triggers` would grow over time.

Reply:
```suggestion
    def purge_completed_triggers(self, session):
```

Reply:
the reason i put `_list` there iss because without it, it could signify "purge complleted triggers _from the database_"

but it's only purging  from the instance attribribute `completed_triggers`.  the actual deletion of triggers from the database is done in TriggerJob.clean_unused

what do you think?